src/omero_zarr/cli.py

```diff
@@ -82,8 +82,9 @@ def _configure(self, parser):
 
         parser.add_argument(
             "--bf",
-            action="store_true",
-            help="Use bioformats2raw to read from managed repo",
+            type=str,
+            help="Full path to bioformats2raw base directory."
```

Comment:
It confused me a _bit_ that this was to e.g. `/opt/bioformats2raw` (despite your instructions). I don't think that needs to be changed, just a heads up.